app/Filament/Resources/ResultResource.php

```diff
@@ -103,6 +108,11 @@ public static function form(Form $form): Form
                                 ->formatStateUsing(function ($state) {
                                     return number_format((float) $state, 0, '.', '').' ms';
                                 }),
+                            Forms\Components\TextInput::make('uploaded_bytes')
+                                ->label('Uploaded bytes')
+                                ->afterStateHydrated(function (TextInput $component, Result $record) {
+                                    $component->state(! blank($record->downloaded_bytes) ? Number::bytesToHuman(bytes: $record->uploaded_bytes, precision: 2) : '');
```

Comment:
You can use the `Number::fileSize()` helper provided by Laravel https://laravel.com/docs/12.x/helpers#method-number-file-size here instead of the custom one.